nncf/sparsity/layers.py

```diff
@@ -24,6 +24,7 @@ class BinaryMask(nn.Module):
     def __init__(self, size):
         super().__init__()
         self.register_buffer("_binary_mask", torch.ones(size))
+        self.sparsify = True
```

Comment:
Rename to `frozen` and update the logic accordingly

Reply:
@vshampor  Ok, but then class `RBSparsifyingWeight` will have two attributes `sparsify` and `frozen`. It looks redundant.

Reply:
Guess you'll have to rename the `RBSparsifytinWeight`'s `sparsify` attribute to `frozen` as well.

Reply:
Done.